### PR TITLE
Feature#48/search from movie api

### PR DESCRIPTION
**Pull request by q270311**

### CODE REVIEW

File: src/features/movieListSlice.js [hunk 1]
```diff
@@ -22,6 +23,9 @@ const movieListSlice = createSlice({
         fetchDataError: (state) => {
             state.status = "error";
         },
+        setQuery: (state, { payload }) => {
```
Comment (kepkaklaudia) [inline src/features/movieListSlice.js:26]: Remember to add here 
`state.status="loading"; `

File: src/features/movieListSlice.js [hunk 2]
```diff
@@ -22,6 +23,9 @@ const movieListSlice = createSlice({
...
+            state.query = payload.query;
+        },
         goToPage: (state, { payload }) => {
```
Comment (kepkaklaudia) [inline src/features/movieListSlice.js:29]: Remember to add here 
`state.status="loading"; `

File: src/features/moviesList/styled.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { Link } from 'react-router-dom';
 export const Wrapper = styled.div`  
   display: grid;
   grid-template-columns: repeat(auto-fill, minmax(324px, 1fr));
+  grid-template-rows: auto;
```
Comment (kepkaklaudia) [inline src/features/moviesList/styled.js:7]: I don't think it changes anything

**Review by kepkaklaudia — APPROVED** [review]
Great! Good first version 👍

**Review by panijoanna — APPROVED** [review]
Well done. You have added the necessary code to make it work. Please remember for the future to consult what are you doing with your contributors.

File: src/features/movieListSlice.js (hunk 2 above)
Comment (q270311) [reply src/features/movieListSlice.js:30]: added :)

File: src/features/moviesList/styled.js (hunk 3 above)
Comment (q270311) [reply src/features/moviesList/styled.js:7]: hmm... I'll remove it now, but we'll see how it works in production

**Pull request merged**